core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -1273,6 +1273,9 @@ static URL toUpdateCenterCheckUrl(String updateCenterUrl) throws MalformedURLExc
          * @throws IOException if the validation fails
          */
         public void preValidate(DownloadJob job, URL src) throws IOException {
+            if(job.site != null) {
```

Comment:
This is failing `master` because it lacked https://github.com/jenkinsci/jenkins/pull/10214.

Reply:
https://github.com/jenkinsci/jenkins/pull/10338